core/src/main/java/tc/oc/pgm/action/actions/TeamAliasAction.java

```diff
@@ -0,0 +1,45 @@
+package tc.oc.pgm.action.actions;
+
+import tc.oc.pgm.api.feature.FeatureReference;
+import tc.oc.pgm.filters.Filterable;
+import tc.oc.pgm.match.MatchImpl;
+import tc.oc.pgm.match.PartyImpl;
+import tc.oc.pgm.teams.Team;
+import tc.oc.pgm.teams.TeamFactory;
+
+public class TeamAliasAction<T extends Filterable<?>> extends AbstractAction<T> {
```

Comment:
```suggestion
public class TeamAliasAction extends AbstractAction<Party> {
```
If this is the team scoped one, the scope should just be party
This change should greatly simplify the rest of the impl of this class as your methods are not just generic T, but party directly.